docs/src/app/components/pages/components/SelectField/Page.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import CodeExample from '../../../CodeExample';
+import PropTypeDescription from '../../../PropTypeDescription';
+import MarkdownElement from '../../../MarkdownElement';
+
+import selectFieldReadmeText from './README';
+import SelectFieldSimpleExample from './ExampleSimple';
```

Comment:
alright, I'll change this. and the other ones too. and I'll do a rebase. thanks for the feedbacks :+1: